src/styles/main.scss

```diff
@@ -1 +1,59 @@
-// put code here
+$circleSize: 80vmin;
```

Comment:
You could make some more variables and move it to the utils folder, as well as create block folder with circle.scss.
main.scss in that case would contain only imports. Just for your practice 😃